src/rules/trailingCommaRule.ts

```diff
@@ -20,11 +20,98 @@ import * as ts from "typescript";
 
 import * as Lint from "../index";
 
+type OptionValue = "always" | "never" | "ignore";
+
+interface CustomOptionValue {
+    arrays?: OptionValue;
```

Comment:
no need to make the properties optional, `Object.assign` defaults them all to "ignore"